src/isolate.rs

```diff
@@ -102,6 +102,19 @@ impl Isolate {
         }
     }
 
+    /// Schedules an exception to be thrown when returning to JavaScript.
+    ///
+    /// When an exception has been scheduled it is illegal to invoke any
+    /// JavaScript operation; the caller must return immediately and only after
+    /// the exception has been handled does it become legal to invoke JavaScript
+    /// operations.
```

Comment:
Does this violate Rust safety? Would it be possible to make this function private or something?